pkg/formatter/formatter.go

```diff
@@ -24,20 +24,17 @@ import (
 	"github.com/chef/chef-analyze/pkg/reporting"
 )
 
-// TODO @afiune make this configurable
-const AnalyzeCacheDir = ".analyze-cache"
-
 // TODO different output depending on flags or TTY?
-func PrintCookbooksReportSummary(records []*reporting.CookbookRecord) {
+func MakeCookbooksReportSummary(records []*reporting.CookbookRecord) string {
 	if len(records) == 0 {
 		// TODO @afiune what is the right text here?
 		fmt.Println("There are no cookbook records to generate a report")
-		return
+		return ""
```

Comment:
I think we should return the string `"There are no cookbook records to generate a report"` instead and delete line `31`
```suggestion
		return "There are no cookbook records to generate a report"
```